crane/container_test.go

```diff
@@ -299,3 +300,11 @@ func TestOptBoolYAML(t *testing.T) {
 	err := yaml.Unmarshal([]byte("OptBool: notaboolean"), &wrapper)
 	assert.Error(t, err)
 }
+
+func TestBuildArgs(t *testing.T) {
+	var c *container
+	// Simple case
```

Comment:
To be honest... I copied `TestActualVolume`. It's simple solution of checking the simplest scenario.
If it's an issue - I'll remove the test.